src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -106,6 +107,15 @@ private IVsStatusbar VsStatusBar
             }
         }
 
+        private async Task<IVsStatusbar> VsStatusBarAsync()
```

Comment:
naming: The method name should probably start with `Get`,

Reply:
Sure, I renamed it.

Reply:
Sorry, I mistook with something else. Just addressed.